lms/templates/admin/macros.jinja

```diff
@@ -63,6 +63,16 @@
     {{ form_text_field(request, "Token URL", "token_url", lti_registration.token_url) }}
 {% endmacro %}
 
+
+{% macro created_updated_fields(model) %}
+    <fieldset class="box">
+    {{ disabled_text_field("Created", model.created) }}
+    {% if model.updated %}
```

Comment:
Some older models don't have `updated`

Reply:
Is the `fieldset` here creating the secondary inner box around this? I think it might be better to let it blend in with the other fields, rather than a box in a box etc.

Reply:
This looks nicer now